ra2ce/network/graph_files/network_file.py

```diff
@@ -28,7 +28,12 @@ def read_graph(self, folder: Path) -> None:
         _file = folder.joinpath(self.name)
         if _file and _file.is_file():
             self.folder = folder
-            self.graph = read_feather(self.file)
+            if self.name.endswith(".feather"):
+                self.graph = read_feather(_file)
+            elif self.name.endswith(".gpkg"):
```

Comment:
Perhaps it's the same, but "semantically" speaking, this is better (we are handling of paths, not strings).
```suggestion
            if _file.suffix == ".feather":
                self.graph = read_feather(_file)
            elif _file.suffix == ".gpkg":
```

Reply:
Adjusted